compoments/report-repair/address.js

```diff
@@ -48,6 +48,9 @@ const Address = ({handleChange, values}) => {
   }
 
   return <div className="govuk-grid-row" data-cy="address">
+    <header>
+      <title>Select an address- Housing repairs</title>
```

Comment:
The second part of the title, i.e. `Housing repairs`, is the service name and is repeated for each component, including existing components, i.e. [landing page/initial service page](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/main/pages/_app.js#L25)

Extract the service title to be globally available (a const or variable) and use for all titles which will reduce duplication and increase maintainability (DRY principle).

I feel the page titles and service title should match. It would be best to ask UX designer to confirm they wanted these to differ and if they are to be consist then ask PO what the service title should be.